src/item.rs

```diff
@@ -7,7 +7,7 @@ pub(crate) enum Item<'src> {
   Assignment(Assignment<'src>),
   Comment(&'src str),
   Import {
-    absolute: Option<PathBuf>,
+    absolute_paths: Vec<PathBuf>,
```

Comment:
I would actually just keep this called `absolute`. It's a vec of paths, so we know that they're paths and there's potentially more than one of them.

Reply:
My argument for changing it is that "absolute" is a pretty general word out of context, so it's hard to guess what it might mean when you see it as a random property access somewhere in code. I don't feel super strongly about it however.